playbookutils.json

```diff
@@ -0,0 +1,521 @@
+    "product_vendor": "Splunk",
+    "logo": "logo_splunk.svg",
+    "logo_dark": "logo_splunk_dark.svg",
+    "product_name": "SOAR",
+    "python_version": "3",
+    "product_version_regex": ".*",
+    "publisher": "Splunk Community",
+    "contributors": [
+        {
+            "name": "Brandan Neises"
+        }
+    ],
+    "license": "Copyright (c) 2022 Splunk Inc.",
+    "app_version": "1.1.0",
+    "utctime_updated": "2021-03-16T15:31:00.718730Z",
+    "package_name": "phantom_playbookutils",
+    "main_module": "playbookutils_connector.py",
+    "min_phantom_version": "5.2.0",
+    "app_wizard_version": "1.0.0",
+    "fips_compliant": false,
+    "pip_dependencies": {
+        "wheel": [
+            {
+                "module": "anytree",
+                "input_file": "wheels/shared/anytree-2.8.0-py2.py3-none-any.whl"
+            },
+            {
+                "module": "beautifulsoup4",
+                "input_file": "wheels/py3/beautifulsoup4-4.9.1-py3-none-any.whl"
+            },
+            {
+                "module": "certifi",
+                "input_file": "wheels/shared/certifi-2021.10.8-py2.py3-none-any.whl"
+            },
+            {
+                "module": "chardet",
+                "input_file": "wheels/shared/chardet-3.0.4-py2.py3-none-any.whl"
+            },
+            {
+                "module": "idna",
+                "input_file": "wheels/shared/idna-2.10-py2.py3-none-any.whl"
+            },
+            {
+                "module": "requests",
+                "input_file": "wheels/shared/requests-2.25.0-py2.py3-none-any.whl"
+            },
+            {
+                "module": "six",
+                "input_file": "wheels/shared/six-1.16.0-py2.py3-none-any.whl"
+            },
+            {
+                "module": "soupsieve",
+                "input_file": "wheels/py3/soupsieve-2.3.2.post1-py3-none-any.whl"
+            },
+            {
+                "module": "urllib3",
+                "input_file": "wheels/shared/urllib3-1.26.9-py2.py3-none-any.whl"
+            }
+        ]
+    },
+    "configuration": {},
+    "actions": [
+        {
+            "action": "test connectivity",
+            "identifier": "test_connectivity",
+            "description": "Validate the asset configuration for connectivity using supplied configuration",
+            "type": "test",
+            "read_only": true,
+            "parameters": {},
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "get playbook tree",
+            "identifier": "get_playbook_tree",
+            "description": "Get details about the parent/child relationships of playbooks and actions",
+            "type": "generic",
+            "read_only": true,
+            "parameters": {
+                "playbook_run_id": {
+                    "description": "Playbook run ID of a playbook in the tree that is to be retrieved (Default to current PB run if not provided)",
```

Comment:
Should we also explain what's considered `current PB` here?
or we should put it in the README.

Reply:
Hello @phantom-bneises,
Could you please look into this comment?

Reply:
I've extended the action documentation to include additional details in the "verbose" section about this and reworded the parameter description here. Please let me know if that makes sense.